.gitignore

```diff
@@ -41,3 +41,6 @@ release.properties
 # Antlr
 *.tokens
 *.interp
+
+# webapp build
+backend/src/main/resources/META-INF
```

Comment:
When a new Quarkus app is generated with https://code.quarkus.io/, the `/src/main/resources/META-INF/resources` contains an `index.html` file which can be displayed in a browser.

This raises two questions:
- Is that OK to add this path to `.gitignore`?
- What is difference between `/src/main/webapp` and `/src/main/resources/META-INF/resources`? Why use one or another?

Reply:
`/src/main/webapp` is holding the source code of the app, while `/src/main/resources/META-INF/resources` will hold the compiled and ready to use html/js/css code which gets automatically served by quarkus.

Reply:
I see, thanks for the explanation!